Detections/QualysVM/HighNumberofVulnDetected.yaml

```diff
@@ -0,0 +1,27 @@
+id: be52662c-3b23-435a-a6fa-f39bdfc849e6
+name: High Number of Urgent Vulnerabilities Detected
+description: |
+  'This Creates an incident when a host has a high number of Urgent, severity 5, vulnerabilities detected.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: CustomConnector
```

Comment:
This is not entirely clear and so I will update documentation, but the connectorId maps from the id in your function app - /Qualys VM/QualysVM_API_FunctionApp.json - specifically the first line -

"id": "QualysVulnerabilityManagement",

and later for dataTypes = "name": "QualysHostDetection_CL",

- connectorId: QualysVulnerabilityManagement
dataTypes:
- QualysHostDetection_CL

Reply:
corrections have been made and pushed.